lib/apollo-federation/tracing.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module ApolloFederation
+  module Tracing
+    KEY = :ftv1
+
+    def self.use(schema)
```

Comment:
Mind wrapping these in a `class << self`?

Reply:
do you have a preference between `class << self` vs `extend self`?

Reply:
lol never mind ... rubocop enforces `module_function`